pandas/io/formats/style.py

```diff
@@ -159,7 +164,8 @@ def __init__(
         self.index = data.index
         self.columns = data.columns
 
-        self.uuid = uuid
+        self.uuid_len = uuid_len if uuid_len < 33 else 32
```

Comment:
i guess this should raise if its not an integer and must be > 0; pls add a test for this